resque-scheduler.gemspec

```diff
@@ -55,6 +55,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'mono_logger', '~> 1.0'
   spec.add_runtime_dependency 'redis', '>= 3.3'
-  spec.add_runtime_dependency 'resque', '~> 1.26'
+  spec.add_runtime_dependency 'resque', '>= 1.26', '<= 2.0.0'
```

Comment:
```suggestion
  spec.add_runtime_dependency 'resque', '>= 1.26'
```

Reply:
Major version bumps are typically known to break backwards compatibility, but in this case 2.0.0 does not.  It's possible for future 2.0.1 to actually be a breaking change, so I erred on the side of caution.

Reply:
>= 1.26 would also allow future 3.0.0 which is most likely not what we want.

Reply:
To be fair, by my own logic a major version bump typically contains breaking changes.
Thus since 2.0.0 did not break resque-scheduler, 2.x.x should not either.

In that case, we should change it to `spec.add_runtime_dependency` 'resque', '>= 1.26', '< 3'`

Reply:
Ahh, okay.  Apologies for resolving conversation.  I've changed as per request.

Reply:
No need to apologize. Your logic makes sense and it is what most projects do. We were being doing that for years in Rails but we recently changed since no release is a breaking change with the previous release.

Reply:
Thank you!  Resolving then.